fdbserver/EncryptionOpsUtils.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * EncryptionOpUtils.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This file implements the functions defined in EncryptionOpUtils.h
+
+#include "fdbserver/EncryptionOpsUtils.h"
+#include "fdbserver/Knobs.h"
+
+#include "flow/Trace.h"
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+bool isEncryptionOpSupported(EncryptOperationType operation_type, ClientDBInfo info) {
```

Comment:
I would prefer to keep it in the cpp file because if decide to change logic there (which will probably happen soon) compile times will increase since this file is included in multiple places

Reply:
Getting it inlined will be more performant, given that the function is being called many times in critical path. Compile time is a lesser issue IMO.

Reply:
Agreed. Also, the routine is simple and contained, @sfc-gh-nwijetunga , can you please move the function definition to the header? 
Thanks